byteverse_24/src/app/components/AboutIamge.jsx

```diff
@@ -0,0 +1,19 @@
+import Image from "next/image";
+import React from "react";
+
+const AboutIamge = () => {
+  return (
+    <div>
+      <Image
+        src="/byteverse/aboutus.png"
+        alt="About US Logo"
+        className="dark:invert  "
```

Comment:
@dev04sa 
I don't think we would want to invert the colors when a user is in dark mode. For that will alter the color scheme completely. Do remember that most of the users use their phones in dark mode itself.